File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Get and validate user input
    int n;

    do
    {
        n = get_int("Height: "); 
    }
    while (n < 1 || n > 8);

    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= n - i;j++)
        {
        printf(" "); 
        }  
            for (int k = 1; k < i + 1 ;k++)
        {
            printf("#");
            
        }   
        
        
       printf("\n");
     
    }
}

File: mario.py
from cs50 import get_int

def main():
    
    i = get_correct_int()
    for a in range(i):
        for b in range(i-a-1):
            print(" ", end ="")
           
        for c in range(a+1):
            print("#", end="")
        print()
            
def get_correct_int():    
    while True:
        height = get_int("Height: ")
        if height>0 and height<9: 
            break
    return height
main()   
